create_table.lua

```diff
@@ -1,21 +1,23 @@
 box.cfg{listen = 3301, memtx_memory = 10 * 1024^3}
 box.schema.user.grant('guest', 'read,write,execute', 'universe')
 
-box.sql.execute("drop table if exists warehouse;")
-box.sql.execute("create table warehouse ( \
+local sql_execute = box.sql and box.sql.execute or box.execute
```

Comment:
I would split changes on different kinds: one commit for retries limiting, another for the schema changes, another for the box.execute / box.sql.execute changes. However I'll not insist.